app/Http/Controllers/VendorsController.php

```diff
@@ -138,4 +141,21 @@ public function destroy($id)
         return redirect('/vendors')->withSuccess('Successfully Deleted');;
     
     }
+
+    public function queryVendors($query)
+    {   
+        $vendors = Vendors::select('id as value', 'name', 'address', 'contact_person','telephone_one')
+            ->where('name', 'LIKE', '%' . $query . '%')->get();
+        return $vendors;
+    }
+
+    public function ajaxGetPaymentsToPay(Vendors $vendor)
+    {
+        return PaymentReferences::select('*')
+            ->leftJoin('payments', 'payments.payment_reference_id', '=', 'payment_references.id')
+            ->where('payment_references.type', '=', 'payment')
+            ->where('payment_references.customer_id', '=', $customer->id)
+            ->where('payment_references.status', '!=', 'paid')->get();
+    }
```

Comment:
Supposedly to be committed to https://github.com/justinmanigo/HMEZGEB/tree/pvv_zmt_vendors, pero sige lang 🚀